books/free-programming-books-ta.md

```diff
@@ -68,6 +69,12 @@
 * [துவக்க நிலையாளர்களுக்கான JavaScript உரைநிரல்](https://freetamilebooks.com/ebooks/javascript_for_beginner/) - ச. குப்பன் (PDF)
 
 
+### Linux
+
+*[Learn GNU Linux in Tamil - Part 1](https://freetamilebooks.com/ebooks/learn-gnulinux-in-tamil-part1/) - து.நித்யா (PDF)
+*[Learn GNU Linux in Tamil - Part 2](https://freetamilebooks.com/ebooks/learn-gnulinux-in-tamil-part2/) - து.நித்யா (PDF)
```

Comment:
need a space after '*'
Don't invent titles! Use the title given by the resource.